contracts/zero-ex/test/storage_uniqueness_test.ts

```diff
@@ -0,0 +1,34 @@
+import { describe } from '@0x/contracts-test-utils';
+import { promises } from 'fs';
+import { basename, resolve } from 'path';
+
+const { readdir, readFile } = promises;
+
+describe('Storage ID uniqueness test', () => {
```

Comment:
I added tests to detect if storage IDs aren't unique. This works so long as we follow the basic pattern of the storage library files.